.env.local.default

```diff
@@ -71,6 +71,8 @@ SERVER_HOST=localhost
 SERVER_PORT=3030
 SERVER_URL=https://localhost:3030
 CORS_SERVER_PORT=3029
+# Switch to `true` to disable logging (useful for unit tests)
+LOG_DISABLE=false
```

Comment:
should be `DISABLE_LOG` not `LOG_DISABLE`

Reply:
Fixed.